src/util/system.h

```diff
@@ -62,6 +62,7 @@ int64_t GetStartupTime();
 static const int64_t DEFAULT_POW_CACHE_SIZE = 1000000;
 static const int DEFAULT_MAX_LOAD_SIZE = 720;
 static bool DEFAULT_VALIDATE_POW_CACHE = false;
+static std::string DEFAULT_IPFS_SERVICE_URL = "https://ipfsm.raptoreum.com/";
```

Comment:
Is this the correct URL?  I get a 404.

Reply:
this is more of an API / get endpoint I think.
Try getting it with `https://ipfsm.raptoreum.com/get/SOME_CID`
Here 2 examples. It will download a .gif and .plain (txt) if you run in the browser. You can `curl -L` the second one to get the text in the terminal for testing.
`https://ipfsm.raptoreum.com/get/QmWKJUMotKNz88E4DtyF6YyWKwMmtpkNXSXKHNGimPyEmg`
`https://ipfsm.raptoreum.com/get/QmW2ccLPSDCu3sWeU5QVrMSyRjd7aoKstR1PaG4aWQ2qUf`

Reply:
Thanks - I am not familiar with the API, so I just wanted to double check.